app/controllers/student_requests_controller.rb

```diff
@@ -58,7 +58,7 @@ def destroy
   end
   
   def adminview
-    @allAdminStates = [StudentRequest::APPROVED_STATE, StudentRequest::REJECTED_STATE, StudentRequest::HOLD_STATE]
+    @allAdminStates = [" ",StudentRequest::APPROVED_STATE, StudentRequest::REJECTED_STATE, StudentRequest::HOLD_STATE]
```

Comment:
Instead of adding a empty state and handling this here, I think it would be more better if you can take the default value in options as first to show. That will require a change in select tag in view.